test/e2e/adot_test.go

```diff
@@ -66,11 +114,23 @@ func runCuratedPackagesAdotInstallSimpleFlow(test *framework.ClusterE2ETest) {
 func runCuratedPackagesAdotInstallUpdateFlow(test *framework.ClusterE2ETest) {
 	test.WithCluster(func(test *framework.ClusterE2ETest) {
 		test.CreateNamespace(adotTargetNamespace)
-		test.InstallCuratedPackage("adot", adotPackagePrefix+"-"+adotPackageName,
+		test.InstallCuratedPackage(adotPackageName, adotPackagePrefix+"-"+adotPackageName,
 			kubeconfig.FromClusterName(test.ClusterName), adotTargetNamespace,
 			"--set mode=deployment")
 		test.VerifyAdotPackageInstalled(adotPackagePrefix+"-"+adotPackageName, adotTargetNamespace)
 		test.VerifyAdotPackageDeploymentUpdated(adotPackagePrefix+"-"+adotPackageName, adotTargetNamespace)
 		test.VerifyAdotPackageDaemonSetUpdated(adotPackagePrefix+"-"+adotPackageName, adotTargetNamespace)
 	})
 }
+
+func runCuratedPackagesAdotInstallTinkerbellSingleNodeFlow(test *framework.ClusterE2ETest) {
```

Comment:
Should this type of function go in cluster.go? It seems to be mostly generic function calls more slated torwards tinkerbell with the only specific adot related to call a function test. Maybe having a tinkerbell setup and tinkerbell teardown is better. Especially if we want to test other flows

Reply:
Please see https://github.com/aws/eks-anywhere/issues/4336